### PR TITLE
Fixing Identity 5020,5017,5015,5029

### PR DESCRIPTION
**Pull request by DMHP**
Starting tenant flow to read the updated changes in registry

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/impl/UserInfoJSONResponseBuilder.java [hunk 1]
```diff
@@ -60,10 +64,18 @@ public String getResponseString(OAuth2TokenValidationResponseDTO tokenResponse)
             throws UserInfoEndpointException {
         Resource resource = null;
         try {
+            PrivilegedCarbonContext.startTenantFlow();
+            PrivilegedCarbonContext carbonContext = PrivilegedCarbonContext.getThreadLocalCarbonContext();
+            int tenantId = IdentityTenantUtil.getTenantIdOfUser(tokenResponse.getAuthorizedUser());
```
Comment (darshanasbg) [inline components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/impl/UserInfoJSONResponseBuilder.java]: Can we assume we get tenant qualified username in all cases like federated users, or saas apps?

Comment (darshanasbg) [inline components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/user/impl/UserInfoJSONResponseBuilder.java]: ie. It should get the tenant from the application (client id) not from the user

**Comment by mefarazath** [comment]
Changes of this PR after addressing the comments is merged with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/202. Therefore we can close this PR now.

**Comment by kasungayan** [comment]
Changes of this PR after addressing the comments is merged with #202. Therefore closing this PR now.

**Pull request closed without merging**